commit/operator/operator_noLUT.c

```diff
@@ -3,8 +3,8 @@
 
 // number of THREADS
 #ifdef nTHREADS
-    #if (nTHREADS<1 || nTHREADS>255)
-    #error "nTHREADS" must be in the range 1..255
+    #if (nTHREADS<0 || nTHREADS>255)
```

Comment:
If `nTHREADS==0` no threads are created (pthread_create). 
This function is not called if `nTHREADS==0` and if it does, it should show an error.
The condition in the `if` should be as before